Casks/chef-workstation.rb

```diff
@@ -0,0 +1,45 @@
+cask 'chef-workstation' do
+  version '0.17.5'
+  sha256 'e9b9165a80222900a1e421f7ad72d2f8252bc0f46175162e43b8cf271b2d2207'
+
+  url "https://packages.chef.io/files/stable/chef-workstation/#{version}/mac_os_x/10.13/chef-workstation-#{version}-1.dmg"
+  name 'Chef Workstation'
```

Comment:
```suggestion
  url "https://packages.chef.io/files/stable/chef-workstation/#{version}/mac_os_x/10.13/chef-workstation-#{version}-1.dmg"
  appcast 'https://omnitruck.chef.io/stable/chef-workstation/metadata?p=mac_os_x&pv=10.14&m=x86_64&v=latest'
  name 'Chef Workstation'
```

Fetching the metadata from omnitruck with `&v=latest` will change whenever we do a release. The contents look like:
```
sha1	b5e1cbc4ee79190a4266316d195e06df0e4b9196
sha256	e9b9165a80222900a1e421f7ad72d2f8252bc0f46175162e43b8cf271b2d2207
url	https://packages.chef.io/files/stable/chef-workstation/0.17.5/mac_os_x/10.14/chef-workstation-0.17.5-1.dmg
version	0.17.5
```

Reply:
Please make the change, @marcparadise.